RecoParticleFlow/PFProducer/python/particleFlow_cfi.py

```diff
@@ -70,12 +70,22 @@
     maxEeleOverPout = cms.double(0.2),
     maxDPhiIN = cms.double(0.1)    
     ),
+    electron_protectionsForBadHcal = cms.PSet(
+    full5x5_sigmaIetaIeta = cms.vdouble(0.0126, 0.0457), # EB, EE
+    eInvPInv = cms.vdouble(0.209, 0.132),
+    dEta = cms.vdouble(0.01, 0.02),
+    dPhi = cms.vdouble(0.148, 0.19),
```

Comment:
these could be indented

Reply:
I would agree but the other nested PSets weren't indented, so I did the same for the sake of consistency.